app/views/items/show.html.erb

```diff
@@ -1,35 +1,76 @@
-<h1>Welcome!</h1>
-
-<div class="container">
-  <h2><%= @item.name %></h2>
-  <!-- IMAGES -->
-  <p><%= @item.price %></p>
-  <p><%= @item.category %></p>git
-  <p><%= @item.brand %></p>
-  <p><%= @item.available %></p>
+<div class="container mt-2"> <%# Page title container %>
+  <div class="row"> <%# Row 1 of 2 %>
+    <div class="container"> <%# h1 to take up half page %>
+      <h1><%= @item.name %></h1>
+    </div>
+  </div>
+  <div class="container"> <%# Row 2 of 2 %>
+    <h3>
+     <%= link_to "User Profile", @user, class: "btn btn-outline-primary" %>
+    </h3>
+  </div>
 </div>
 
-<div>
-<%= link_to "Remove", item_path(@item), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" }, class: "ms-3 text-muted" %>
-</div>
+<div class="container mt-3">
+    <!-- IMAGE -->
+    <%= image_tag "#{rand(1..6)}.jpg", class: "rounded float-left" %>
+    <!-- DETAILS -->
+    <ul>
+    <br>
+      <li>
+        <p class="card-text">
+          <% if @item.available == true %>
+            <%= "AVAILABLE" %>
+          <% end %>
+          <% if @item.available == false %>
+            <%= "UNAVAILABLE" %>
+          <% end %>
+        </p>
+      </li>
+      <li>
+        <p class="card-text">Price: €<%= @item.price %></p>
+      </li>
+      <li>
+        <p class="card-text">Category: <%= @item.category %></p>
+      </li>
+      <li>
+        <p class="card-text">Brand: <%= @item.brand %></p>
+      </li>
+    </ul>
+
+    <!-- LINK TO EDIT LISTING-->
+    <div>
+     <%= link_to "Edit Listing", item_path(@item), data: { turbo_method: :edit}, class: "ms-3 text-muted" %>
+    </div>
 
+    <!-- LINK TO DELETE LISTING -->
+    <div>
+      <%= link_to "Remove Listing", item_path(@item), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" }, class: "ms-3 text-muted" %>
+    </div>
+</div>
 
 <!-- # FORM FOR RENTAL -->
-<div class="rental inquiry">
-  <h2> Submit a Rental Inquiry <h2>
+<div class="container mt-3">
+  <div class="mx-auto" style="width: 400px;">
+    <h2> Submit a Rental Inquiry <h2>
+  </div>
   <%= simple_form_for [@item, @rental] do |f| %>
-    <%= f.input :start_date %>
-    <%= f.input :end_date %>
-    <%# f.input :price %>
-    <%= f.submit class: "btn btn-primary" %>
+    <div class="form-inputs">
+      <%= f.input :start_date %>
+      <%= f.input :end_date %>
+    </div>
+    <div class="form-actions">
+      <%= f.button :submit, class:"btn btn-primary" %>
+    </div>
   <% end %>
 </div>
 
-<!-- # LINK USER PROFILE -->
 <!-- # LINK TO HOME -->
-<!-- # LINK TO DELETE -->
+<div class="mx-auto" style="width: 200px;">
+  <%= link_to "Home Page", item_path(@item), class: "btn btn-outline-primary" %>
```

Comment:
instead of item_path it should be root_path as item_path(@item) is the show page :)

Reply:
Thank you! Just changed, ready for review :)